vmc/resource_vmc_sddc.go

```diff
@@ -596,18 +601,26 @@ func resourceSddcUpdate(d *schema.ResourceData, m interface{}) error {
 	}
 
 	if d.HasChange("edrs_policy_type") || d.HasChange("enable_edrs") || d.HasChange("min_hosts") || d.HasChange("max_hosts") {
-		edrsPolicyClient := autoscalercluster.NewDefaultEdrsPolicyClient(connectorWrapper)
+
+		SDDCType := d.Get("sddc_type").(string)
```

Comment:
Is SDDCType a constant or exported variable? Wondering why this naming convention is required..